src/shared/components/annotation/OncoKB.tsx

```diff
@@ -170,7 +170,7 @@ export default class OncoKB extends React.Component<IOncoKbProps, {}>
         const geneParam = `entry.1744186665=${hugoSymbol || ''}`;
         const alterationParam = `entry.1671960263=${alteration || ''}`;
         const userParam = `entry.1381123986=${this.props.userEmailAddress || ''}`;
-        const uriParam = `entry.1083850662=${encodeURIComponent(window.location.href)}`;
+        const uriParam = `entry.1083850662=${encodeURIComponent(window.location.href.replace(/navCaseIds=[^&]*/, ''))}`;
```

Comment:
probably better to just build window.location.hostname+window.location.path+window.location.search. but i'll approve and you make change if you want

Reply:
@alisman I'm thinking to keep other parameters after the pound sign

Reply:
i just don't want to have that parameter hardcoded.  i don't think the hash will be important

Reply:
Thanks. Updated the url.